<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleChComb</class>
 <widget class="QWidget" name="SingleChComb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>23</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="singleVoltChk">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>CHX</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nPoreStateBtn">
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="statusTip">
      <string>Blue is pore identify</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ampPlusOrMinusBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="ampDpBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>58</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="keyboardTracking">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> V</string>
     </property>
     <property name="minimum">
      <double>-2.500000000000000</double>
     </property>
     <property name="maximum">
      <double>2.500000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyBtn">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="degatAutoChk">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="degatColorBtn">
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
